### ISSUE TITLE
"Correcting..." spam in log/console 

### ISSUE THREAD

**Issue by TheKoci** [EnigmaticaModpacks/Enigmatica6#2939]
0.5.9

Console spamming config correction. We never modified this config file.

I tried putting latest.log on pastebin but the spam is so huge I will post the one line that is spamming every second

[10Aug2021 20:37:38.421] [Thread-1/WARN] [net.minecraftforge.fml.config.ConfigFileTypeHandler/CONFIG]: Configuration file /home/user01/servers/enigma/config/naturesstarlight-common.toml is not correct. Correcting

This message is being sent every second in console. We had to install DeLogger to make console clear.

**Comment by NillerMedDild** [comment]
This should be fixed in the next update, you should be able to fix it yourself by deleting the `naturesstarlight-common.toml` config :)

**Comment by iarspider** [comment]
Still happening in 1.1.2, deleting that file did not help

**Comment by Quezler** [comment]
Still present in 1.3.0

**Comment by iarspider** [comment]
The issue is in Nature's Starlight: https://github.com/Ellpeck/NaturesStarlight/issues/1
The only solution for now is to use a preview version: https://github.com/Ellpeck/NaturesStarlight/issues/1#issuecomment-899026907

**Comment by Quezler** [comment]
**TLDR** since we can only use versions released on curseforge we'll have to wait for @Ellpeck to upload a fixed one there:
![Screen Shot 2022-05-27 at 14 45 43](https://user-images.githubusercontent.com/3179271/170702108-21ff96fc-51be-4831-81bf-1cbe1af6bdbb.png)

**Comment by iarspider** [comment]
... or get a permission from @Ellpeck to redistribute the preview version and put it "overrides" (I guess this is how it works)

**Comment by Discomanco** [comment]
> (I guess this is how it works)

Sadly not how it works. Mods that are in the `overrides` folder, are 3rd party mods (so not hosted on CurseForge) that are allowed to be distributed by CurseForge
